response.go

```diff
@@ -134,6 +134,14 @@ func (r *Response) IsError() bool {
 	return r.StatusCode() > 399
 }
 
+// Close method closes the inner HTTP response body.
+func (r *Response) Close() error {
```

Comment:
@divinerapier `r.RawBody()` returns value only if Resty user uses https://godoc.org/github.com/go-resty/resty#Request.SetDoNotParseResponse otherwise it will be always `nil` since resty fully reads the response body and closes it. 

Can you please explain or I might be failing to understand something?